### DISCUSSION TITLE
Connect ESP32 to phone without application over Bluetooth classic

### DISCUSSION THREAD

**Discussion by dharmikP17** [espressif/arduino-esp32#8198] [Q&A]
I am using ESP32 dev module with Arduino IDE. 
ESP version : 2.0.5 Arduino version : 2.0.2

My goal is to get the MAC address of the devices(phones to be specific). I can get that easily if I can connect to EPS32 but I could find a way to do this without using application. I can not use or create application as this will be used for a number of people and devices. Is it possible to do this ?

If not, than is it possible to do the same using paring functionality ? I do get a callback in ESP32 when new device tries to pair however it will stay paired in the device. Can I remove my EPS32 from "paired devices" from phone automatically (from ESP32) ? Because if I will not do that than I will not get the callback next time as it work only when we pair first time.

In other word I want to get the MAC address of the device using options from phone's settings.

**Comment by mrengineer7777** [comment]
> Can I remove my EPS32 from "paired devices" from phone automatically (from ESP32)?

I doubt it.  That would be a severe security risk if external devices could delete Bluetooth settings.

Try using SoftAP (AP mode) instead of Bluetooth. That way the ESP32 knows the phone's MAC address when it connects.  The ESP32 can turn off SoftAP to force a disconnect.
